webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {path: path.join(__dirname, "/dist"),   
+  filename: "bundle.js"
+ },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: 'babel-loader',
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({ template: './src/index.html' }),
+  ],
+};
```

Comment:
Add single empty line on your files which has this git warning⛔
![image](https://user-images.githubusercontent.com/58233753/99066800-3b0c4b80-25b2-11eb-8ca1-5a556379cf3b.png)

Reply:
Done.